roll.py

```diff
@@ -282,6 +288,36 @@ def load_config() -> Config:
         config.l2_engine_rpc = "http://127.0.0.1:8545"
         config.l2_engine_rpc_listen_port = 8545
 
+    # Update endpoints if remote deployment
+    if config.l1_node_remote_ip is not None:
+        setattr(
+            config,
+            "l1_rpc",
+            config.l1_rpc.replace("127.0.0.1", config.l1_node_remote_ip)
+        )
+        setattr(
+            config,
+            "l1_rpc_for_node",
+            config.l1_rpc_for_node.replace("127.0.0.1", config.l1_node_remote_ip)
+        )
+    if config.l2_engine_remote_ip is not None:
+        setattr(
+            config,
+            "l2_engine_rpc",
+            config.l2_engine_rpc.replace("127.0.0.1", config.l2_engine_remote_ip)
+        )
+        setattr(
+            config,
+            "l2_engine_authrpc",
+            config.l2_engine_authrpc.replace("127.0.0.1", config.l2_engine_remote_ip)
+        )
+    if config.l2_sequencer_remote_ip is not None:
+        setattr(
+            config,
+            "l2_node_rpc",
+            config.l2_node_rpc.replace("127.0.0.1", config.l2_sequencer_remote_ip)
+        )
+
```

Comment:
Should probably be `config.l1_rpc = ....`

See above command on `config.py` on using those but splitting them between address and port.

Not super important for now, I can do it if it's convenient.